src/common/ActionButtons/SecondaryButton.tsx

```diff
@@ -23,6 +23,7 @@ const SecondaryButton: FC<SecondaryButtonProps> = props => {
 			payload={[button]}
 			action={action}
 			customIcon={customIcon}
+			noIcon={!customIcon}
```

Comment:
this seems like unnecessary prop drilling, any not to reason to just check !customIcon directly in ActionButton? (same for PrimaryButton.tsx)

Reply:
Actually no, because the ActionButton, by default add Link icon to any web_url button. In order to avoid that, we need to pass a customIcon or define noIcon as true on top.
But I also understand that is maybe a bit confusing, I will try to improve it.

Reply:
done